packages/react/src/users/hooks/types/useUserContacts.types.ts

```diff
@@ -0,0 +1,6 @@
+import type { Config } from '@farfetch/blackout-client';
+
+export type UseUserContactsOptions = {
```

Comment:
I think it is better to have one options type for each hook as they might diverge in the future.

Reply:
👍  Changed it